source/assets/scripts/main.js

```diff
@@ -158,6 +160,7 @@ function addTasksToDocument(savedTasks) {
 /**
  * Takes in an array of tasks, converts it to a string, and then
```

Comment:
Again, add a more general description at. beginning. Essentially we just save the task to todoListDB in localStorage